tasks/main.yml

```diff
@@ -1,12 +1,15 @@
 ---
-- name: Set version specific variables
+- name: Set platform/version specific variables
   include_vars: "{{ item }}"
-  with_first_found:
-    - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
-    - "{{ ansible_distribution }}.yml"
-    - "{{ ansible_os_family }}.yml"
-    - "default.yml"
+  loop:
+    - "{{ role_path }}/vars/default.yml"
```

Comment:
This repeated `{{ role_path }}/vars` is not very aestehetical. Can you use the same or similar approach as in https://github.com/linux-system-roles/nbde_server/pull/9/files#diff-2444ad0870f91f17ca6c2a5e96b26823R8 ?

Reply:
Hmmm, is it ok not to follow https://github.com/oasis-roles/meta_standards/#supporting-multiple-distributions-and-versions?

Reply:
This changes the "first found" semantics.  This change should not do that.  This change should:
* use role_path to use absolute path
* use the lookup function form instead of `with_first_found`
* use `ansible_facts` instead of `ansible_varname`

Reply:
> This changes the "first found" semantics. This change should not do that. 

Ouch! I missed that.

Reply:
> Hmmm, is it ok not to follow https://github.com/oasis-roles/meta_standards/#supporting-multiple-distributions-and-versions?

Yes.  That is only for new code that can be designed so that you do not have to use "first found" semantics.  For existing code, to be safe, we have to preserve "first found" semantics.